### PR TITLE
Fixes click dragging clothes on the ground

### LINKED ISSUE #24156: ctrl clicking clothes to pull them doesnt work anymore
#### Description of issue
You cannot ctrl click clothing on the ground to drag them behind you, you used to be able to do this.

#### Difference between expected and actual behavior
ctrl click something to initiate a pull and it working
doesnt work.

#### Steps to reproduce
ctrl click a jumpsuit on the ground and try to move with it behind you.

#### Specific information for locating

works for everything else except pieces of clothing, hats, suits, boots, etc.
obj/item/clothing items are the problem

#### Length of time in which bug has been known to occur

This works on a 3 month old localtest.

#### Server revision

3cdb4071343acaf14e88456efd4c089b5bcc1f25

### PR DESCRIPTION
**Pull request by Dacendeth**
Can once again ctrl click clothes on the ground to drag them without losing the accessory removal feature on worn clothes.
Fixes #24156
[bugfix]

:cl:
 * bugfix: You can once again ctrl click clothes on the ground to drag them.

### CODE REVIEW

File: code/modules/clothing/clothing.dm [hunk 1]
```diff
@@ -22,6 +22,8 @@
 	..()
 	
 /obj/item/clothing/CtrlClick(var/mob/user)
+	if(istype(loc,/turf))
+	return ..()
```
Comment (Kurfursten) [inline code/modules/clothing/clothing.dm]: You have to indent this!

**Review by Kurfursten — CHANGES_REQUESTED** [review]

File: code/modules/clothing/clothing.dm (hunk 1 above)
Comment (Dacendeth) [reply code/modules/clothing/clothing.dm]: What's wrong exactly, this is how i indented it locally when i tested it, everything else i tried said something about inconsistent indentation

Comment (Dacendeth) [reply code/modules/clothing/clothing.dm]: okay I think I might see what I need to do looking at other code.

**Review by Kurfursten — APPROVED** [review]

File: code/modules/clothing/clothing.dm [hunk 2]
```diff
@@ -22,6 +22,8 @@
 	..()
 	
 /obj/item/clothing/CtrlClick(var/mob/user)
+	if(istype(loc,/turf))
```
Comment (Exxion) [inline code/modules/clothing/clothing.dm]:
```suggestion
	if(isturf(loc))
```

**Comment by Kurfursten** [comment]
They both work but exxion is right, using the macro is better for a few minor reasons

**Comment by Exxion** [comment]
It's not a macro
That one's a built-in proc
It doesn't make a difference in a sane codebase but it works differently internally

**Pull request merged**